EtherealCereal/EtherWrapper.h

```diff
@@ -48,6 +48,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSString *)signMessage:(NSData *)message withKey:(NSData *)privateKey NS_SWIFT_NAME(sign(message:with:));
 
+/**
+ Sign a message/transaction with a given private key without hashing (always raw data).
+
+ @param message The message or transaction to be signed.
+ @param privateKey The raw data private key from the user. Should be kept safe at all costs.
+ @return A 130 character hexadecimal string. This is the signature for a given message. If you have this and the original message, you can recover an address. Use it to guaratee the message originator.
```

Comment:
Worth adding tests around: 
- The returned string being 130 characters?
- Being able to recover the address from a message?